lib/Graph.dart

```diff
@@ -21,7 +21,7 @@ class Graph {
 
   void addNodes(List<Node> nodes) => nodes.forEach((it) => addNode(it));
 
-  void removeNode(Node node) {
+  void removeNode(Node? node) {
```

Comment:
❓  why this one is nullable?
looks to me if you want to remove `Node`, you want to remove `Node`, but not `null` 😄